lottie/src/main/java/com/airbnb/lottie/manager/ImageAssetManager.java

```diff
@@ -102,6 +102,25 @@ public void setDelegate(@Nullable ImageAssetDelegate assetDelegate) {
       return putBitmap(id, bitmap);
     }
 
+    //if no images folder given assume drawable res
+    if (TextUtils.isEmpty(imagesFolder)) {
+      try
+      {
+        int drawableId = context.getResources().getIdentifier(filename, "drawable", context.getPackageName())
+        if(drawableId != -1)
+        {
+          //we dont pass "opts" we rely on system scaling drawables
+          bitmap = BitmapFactory.decodeResource(context.getResources(), drawableId);
+          bitmap = Utils.resizeBitmapIfNeeded(bitmap, asset.getWidth(), asset.getHeight());
+          return putBitmap(id, bitmap);
+        }
+      }
+      catch (Exception e) 
```

Comment:
What specific exceptions are you catching here and for which lines?

Reply:
Leading brace is on the same line as the catch
`} catch (SpecificException e) {`

Reply:
fixed.
was just copying the catch from load asset. changed it to IOException